examples/mean-reversion.go

```diff
@@ -0,0 +1,241 @@
+package main
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"time"
+
+	"github.com/alpacahq/alpaca-trade-api-go/alpaca"
+	"github.com/alpacahq/alpaca-trade-api-go/common"
+	"github.com/shopspring/decimal"
+)
+
+type alpacaClientContainer struct {
+	client         *alpaca.Client
+	runningAverage float64
+	lastOrder      string
+	timeToClose    float64
+	stock          string
```

Comment:
How do you set this stock?

Reply:
oh hard coded below. Do you want to take command line argument and make this completely runnable code? Currently you need to modify `init()` to even start